src/views/virtualmachines/list/VirtualMachinesList.tsx

```diff
@@ -61,227 +61,260 @@ import '@kubevirt-utils/styles/list-managment-group.scss';
 import './VirtualMachinesList.scss';
 
 type VirtualMachinesListProps = {
+  cluster?: string;
   kind: string;
   namespace: string;
 } & RefAttributes<{ onFilterChange: OnFilterChange } | null>;
 
```

Comment:
Added cluster prop, which is being handed off from useNavigationLocationParam hook.